Swiften/MIX/MIX.h

```diff
@@ -59,11 +61,22 @@ namespace Swift {
             */
             virtual void updatePreferences(Form::ref form) = 0;
 
+            /**
+            * Register nick with MIX Channel.
+            */
+            virtual void registerNick(const std::string &nick) = 0;
```

Comment:
I am not sure whether disco query needs to be done before (May be done in controllers ?)